src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -317,7 +317,7 @@ public static bool IsInOutContext(this ExpressionSyntax expression)
         }
 
         public static bool IsInRefContext(this ExpressionSyntax expression)
-            => expression.IsParentKind(SyntaxKind.RefExpression) ||
+            => expression?.IsParentKind(SyntaxKind.RefExpression) == true ||
```

Comment:
All of our `IsParentKind` extensions have a null check upfront and we avoid null checks at all call sites. Please revert this change.